spa/src/components/pageEditor/editInterface/pageStyles/PageStyles.js

```diff
@@ -36,9 +37,16 @@ function PageStyles({ backToProperties }) {
   };
 
   return (
-    <S.PageStyles>
-      <StylesChooser styles={availableStyles} selected={styles} setSelected={(s) => setStyles(s)} />
-      <S.Buttons>
+    <Root>
+      <EditTabHeader
+        addButtonLabel="Style"
+        onAdd={() => setAddStylesModalOpen(true)}
```

Comment:
Could we re-write `const [addStylesModalOpen, setAddStylesModalOpen] = useState(false);` to use our custom hook `useModal`?
```suggestion
        onAdd={handleOpenAddStylesModal}
```

Reply:
Updated in ec41fbaaee2ab1759f0340e84a2bb635db63314d